hafnian/quantum.py

```diff
@@ -609,40 +609,38 @@ def gen_Qmat_from_graph(A, n_mean):
     return Q
 
 
-def is_valid_cov(cov, hbar=2, sigdigits=6):
+def is_valid_cov(cov, hbar=2, rtol=1e-05, atol=1e-08):
     r""" Checks if the covariance matrix is a valid quantum covariance matrix.
 
     Args:
         cov (array): a covariance matrix
         hbar (float): value of hbar in the uncertainty relation

```

Comment:
Need to add `rtol` and `atol` to docstrings whereever they are added
```suggestion
        hbar (float): value of :math:`\hbar` in the uncertainty relation
        rtol (float): relative tolerance used when checking validity
        atol (float): absolute tolerance used when checking validity
```